### PR TITLE
add support to jsp

### PR DESCRIPTION
**Pull request by aohanhongzhi**
add support to jsp

### CODE REVIEW

File: language.go [hunk 1]
```diff
@@ -490,6 +491,7 @@ func NewDefinedLanguages() *DefinedLanguages {
 			"JAI":                 NewLanguage("JAI", []string{"//"}, [][]string{{"/*", "*/"}}),
 			"Janet":               NewLanguage("Janet", []string{"#"}, [][]string{{"", ""}}),
 			"Java":                NewLanguage("Java", []string{"//"}, [][]string{{"/*", "*/"}}),
+			"Jsp":                 NewLanguage("Jsp", []string{"//"}, [][]string{{"/*", "*/"}}),
```
Comment (hhatto) [inline language.go:494]: Can you change the `Jsp` to an upper case?
```suggestion
			"JSP":                 NewLanguage("JSP", []string{"//"}, [][]string{{"/*", "*/"}}),
```

File: language.go [hunk 2]
```diff
@@ -156,6 +156,7 @@ var Exts = map[string]string{
 	"ipynb":       "Jupyter Notebook",
 	"jai":         "JAI",
 	"java":        "Java",
+	"jsp":         "Jsp",
```
Comment (hhatto) [inline language.go:159]:
```suggestion
	"jsp":         "JSP",
```

**Review by hhatto — CHANGES_REQUESTED** [review]
Thank you for contribution 👍 

I made one comment, can you confirm it?

File: language.go (hunk 2 above)
Comment (aohanhongzhi) [reply language.go]: ok, I have changed the `Jsp` to an upper case

**Review by hhatto — APPROVED** [review]
LGTM 👍 

Thanks

**Pull request merged**